src/doc/trpl/installing-rust.md

```diff
@@ -61,21 +61,89 @@ binary downloads][install-page].
 
 ## Platform support
 
-Oh, we should also mention the officially supported platforms:
-
-* Windows (7 or later, Server 2008 R2)
-* Linux (2.6.18 or later, various distributions), x86 and x86-64
-* OSX 10.7 (Lion) or later, x86 and x86-64
-
-We extensively test Rust on these platforms, and a few others, too, like
-Android. But these are the ones most likely to work, as they have the most
-testing.
-
-Finally, a comment about Windows. Rust considers Windows to be a first-class
-platform upon release, but if we're honest, the Windows experience isn't as
-integrated as the Linux/OS X experience is. We're working on it! If anything
-doesn't work, it is a bug. Please let us know if that happens. Each and every
-commit is tested against Windows just like any other platform.
+Rust supports a wide variety of platforms each of which is organized into one of
+three tiers of support. Each tier brings with it a different set of guarantees,
```

Comment:
This sentence contains 'support' twice, awkward. "The Rust compiler runs on, and compiles to, a great number of platforms, though not all platforms are equally supported. Rust's support levels are organized into three tiers, each with a different set of guarantees." Not sure, tired.